docs/db/changelog/changesets/powerauth-java-server/1.6.x/20231123-operation-user-nullable.xml

```diff
@@ -1,14 +1,14 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
                    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-4.9.xsd">
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-4.20.xsd">
```

Comment:
is this necessary?

Reply:
Already merged. I was missing `columnDataType` in the namespace, but XSD has not been fetched. Anyway, it does not harm.

Reply:
It actually breaks the checksum (I guess with the adding column data type), so yes, this did some harm...